internal/kv/rocksdb/cwrapper/CMakeLists.txt

```diff
@@ -21,9 +21,15 @@ set(CMAKE_CXX_STANDARD 17)
 set(CMAKE_EXPORT_COMPILE_COMMANDS ON)
 
 include( ExternalProject )
-set( ROCKSDB_VERSION "6.15.2" )
-set( ROCKSDB_SOURCE_URL
-    "https://github.com/facebook/rocksdb/archive/v${ROCKSDB_VERSION}.tar.gz")
+if ( APPLE )
+    set( ROCKSDB_VERSION "6.27.3" )
```

Comment:
Please pay attention to the x86 mac ci pass

Reply:
Will check.